src/library/templates/library/preview_content_lib.html

```diff
@@ -0,0 +1,54 @@
+<li class="list-group-item">
+  <div class="row visible-xs-block">
+    <div class="col-xs-6">
+      <p>Campaign: {{ quest.campaign|default:"-" }}</p>
+    </div>
+    <div class="col-xs-6">
+      <p>{{ config.custom_name_for_tag }}s: <small>{{ quest.tags.all|join:", "|default:"-" }}</small></p>
+    </div>
+  </div>
+  <hr class="visible-xs-block">
+
+  <div id="preview-btns-{{ quest.id }}" class="pull-right" style="margin-bottom: 1em;">
```

Comment:
This file appears identical to `quest_manager/preview_content_quests_avail.html`  except for the set of buttons here.  Instead of duplicating all the code here, can we instead introduce some logic that just changes the buttons based on the library context, when on the library tab?

The `quest_manager/buttons.html` already has a lot of logic in it to determine which buttons to present, so might make sense to use that.

There's a reason for this, for example you've already changed this so it's different in ways we don't want it to be different from preview_content_quests_avail.html:

` <p>{{ quest.short_description|default:"No description." }}</p`    
vs   
` <p>{{q.short_description|safe}}</p>`

These lines should be identical.